src/EvenVCO.cpp

```diff
@@ -34,6 +34,7 @@ struct EvenVCO : Module {
 	/** The outputs */
 	/** Whether we are past the pulse width already */
 	bool halfPhase[PORT_MAX_CHANNELS] = {};
+	bool removeDCFromPulse = false;
```

Comment:
If this is in fact closer to EvenVCO's behavior, I'd rather just switch to that instead of maintain worse behavior to make old patches perfectly unchanged. But is it closer? What does the hardware do?

Reply:
I don't have the hardware, but I think (from youtube videos) that it _will_ include DC for PW != 50%. In most scenarios I think you want to remove DC, but if you're modulating it at audio rate you can get some funky results (where you probably don't want DC correction). I think 9 times out of 10 you modulate at LFO speeds where you probably want DC correction

![image](https://user-images.githubusercontent.com/915048/134860161-151803c3-5f2c-4f70-a79d-a7605596d8bb.png)

Reply:
I'll check with my scope when I'm at home.

Reply:
Bit more chatter / context in this thread if it helps:

https://community.vcvrack.com/t/befaco-modules-development-blog/13969/18?u=hemmer

It's a tricky one!

Reply:
![evenvco_scope](https://user-images.githubusercontent.com/915048/135398778-745beceb-0de3-4a40-9620-b7ac3ac6d97a.gif)

@AndrewBelt - looks like the hardware doesn't remove DC. I wonder if because EvenVCO is so widely used it's just simpler to revert the changes and preserve backwards compatibility. Befaco want to do another VCO soon, so maybe then I can discuss / implement DC removal. Do you agree?

(I think this is the only outstanding change, so once we're happy here I can look at V2 builds etc.)

Reply:
Yes, let's match hardware and remove the DC blocker.

Reply:
Done